src/main/java/me/jellysquid/mods/sodium/client/render/pipeline/BlockRenderer.java

```diff
@@ -105,14 +106,14 @@ private void renderQuadList(BlockRenderView world, BlockState state, BlockPos po
                 colorizer = this.blockColors.getColorProvider(state);
             }
 
-            this.renderQuad(world, state, pos, sink, offset, colorizer, quad, light);
+            this.renderQuad(world, state, pos, sink, offset, buffers, colorizer, quad, light);
         }
 
         sink.flush();
     }
 
     private void renderQuad(BlockRenderView world, BlockState state, BlockPos pos, ModelVertexSink sink, Vec3d offset,
-                            BlockColorProvider colorProvider, BakedQuad bakedQuad, QuadLightData light) {
+                            ChunkModelBuffers buffers, BlockColorProvider colorProvider, BakedQuad bakedQuad, QuadLightData light) {
```

Comment:
This should simply pass the `ChunkRenderData.Builder` type directly to avoid `getRenderData()` calls below. We shouldn't need to access anything else on `ChunkModelBuffers`, anyways.